python/GangaLHCb/test/GPI/TestDataFiles.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from Ganga.testlib import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+
+class TestDataFiles(GangaUnitTest):
+
+    @external
+    def testDataFiles(self):
+
+        from Ganga.GPI import DiracFile, config
+
+        print config['LHCb']
+        # LFNs
+        name = 'test.txt'
+        lfn = DiracFile(lfn=name)
+        assert lfn.lfn == name
+        lfn = DiracFile('lfn:'+name)
+        assert lfn.lfn == name
+        lfn = DiracFile('LFN:'+name)
+        assert lfn.lfn == name
+        try:
+            lfn = DiracFile('pfn:'+name)
```

Comment:
this is news (and scares me a little) how can a call to a DiracFile init return a LocalFile?